modules/installation-aws-upi-installation.adoc

```diff
@@ -0,0 +1,187 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws_upi/installing-aws-upi.adoc
+
+[id="installation-aws-upi-cleanup-{context}"]
+= Completing an AWS UPI installation
+
+After you start the {product-title} installation on Amazon Web Service (AWS)
+user-provisioned infrastructure, remove the bootstrap node, reconcile the default
+Machine and MachineSet definitions, and delete unused nodes
+
+.Prerequisites
+
+* Deploy the bootstrap node for a{product-title} cluster on user-provisioned AWS infrastructure.
+* Install the `oc` CLI and log in.
+
+.Procedure
+
+. Delete the bootstrap resources. If you used the CloudFormation template,
+link:https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-console-delete-stack.html[delete its stack]:
++
+----
+$ aws cloudformation delete-stack --stack-name <name> <1>
+----
+<1> `<name>` is the name of your bootstrap stack.
+
+. View the list of machines in the `openshift-machine-api` namespace:
```

Comment:
Seem like the following steps is doing https://github.com/openshift/installer/blob/master/docs/user/aws/install_upi.md#option-1-dynamic-compute-using-machine-api. These steps are not working well, refer to https://bugzilla.redhat.com/show_bug.cgi?id=1697968.

Reply:
I think they're working on it, but I'll track the bug and follow up when it's resolved.